plugins/base/src/test/kotlin/markdown/ParserTest.kt

```diff
@@ -951,6 +951,7 @@ class ParserTest : KDocTest() {
     fun `Multilined Code Block`() {
         val kdoc = """
         | ```kotlin
+        | @Suppress("UNUSED_VARIABLE")
```

Comment:
I think it'd be better to have a separate small test that checks just `@` inside code blocks. In case this particular feature breaks it'd be easier to spot and understand what exactly broke.